examples/only-logs-fluentd/README.md

```diff
@@ -0,0 +1,4 @@
+# Example of chart configuration
+
+## Only collect logs with the fluentd logs engine
+This example shows how you can use collect fluentd logs (no metric, no traces).
```

Comment:
I don't understand "how you can use collect fluentd logs". Can you clarify what is meant here?